rest-api/dao/questionnaire_response_dao.py

```diff
@@ -253,12 +253,18 @@ def _update_participant_summary(
           count_completed_ppi_modules(participant_summary)
 
     if something_changed:
-      first_last_email = (
-          participant_summary.firstName, participant_summary.lastName, participant_summary.email)
-      if not all(first_last_email):
+      first_last = (
```

Comment:
Update the logic in here:

https://github.com/all-of-us/raw-data-repository/blob/3bd458de95d02a3a2bf802089453975df0eca016/rest-api/test/unit_test/dao_test/questionnaire_response_dao_test.py#L210

To verify the change in behavior. You should be able to successfully do first name, last name, email or first name, last name, login phone number.

Reply:
I add a new method test_insert_both_names_and_login_phone_number for this one, please review.